Bio/Align/AlignInfo.py

```diff
@@ -190,23 +190,28 @@ def iupac_consensus(self, ambiguous="X",
         of bases (e.g., if only 1 base differs in an alignment of 100, 
         this will will call an ambiguity). A very conservative approach.
 
-        iupac_consensus would output the following consensus 
-        given an alignment with these 4 sequences:
-        seq1 TATCGACCTCATCG
-        seq2 TAAGATTGCGATCG
-        seq3 TAAGATTGGTATCG
-        seq4 TAAGATTGATATCG
-        -------------------
-        con  TAWSRWYSNBATCG
-
-        dumb_consensus would have output the following consensus 
-        given an alignment with these 4 sequences: 
-        seq1 TATCGACCTCATCG
-        seq2 TAAGATTGCGATCG
-        seq3 TAAGATTGGTATCG
-        seq4 TAAGATTGATATCG
-        -------------------
-        con  TAAGATTGNNATCG
+        For a test alignment with 4 sequences, 
+        iupac_consensus would output the following consensus:
+
+        >>> from Bio import AlignIO
+        >>> alignment = AlignIO.read('../Tests/GFF/multi_1.fna', 'fasta')
```

Comment:
You can leave out the ``../Tests/`` bit as we run the doctests from the ``Tests/`` directory.

Reply:
I was just asking about that.